chart/templates/environment-secrets.yaml

```diff
@@ -0,0 +1,59 @@
+{{- if gt .Capabilities.KubeVersion.Minor "26" }}
+# NEW EKS CLUSTER
+---
+apiVersion: external-secrets.io/v1beta1
+kind: SecretStore
+metadata:
+  name: vets_api_secretstore
+  annotations:
+    argocd.argoproj.io/hook: PreSync
+    argocd.argoproj.io/sync-wave: "-4"
+spec:
+  provider:
+    aws:
+      service: ParameterStore
+      region: us-gov-west-1
+      auth:
+        jwt:
+          serviceAccountRef:
+            name: vets-api
+
+---
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: common_environment_vars
+  annotations:
+    argocd.argoproj.io/hook: PreSync
+    argocd.argoproj.io/sync-wave: "-3"
+spec:
+  secretStoreRef:
+    name: vets_api_secretstore
+    kind: SecretStore
+  data:
+  {{- range $key, $value := .Values.parameters }}
+    - secretKey: {{ $key }}
+      remoteRef:
+        key: {{ $.Values.ssmPath }}{{ $value }}
+  {{- end }}
+
+{{- else }}
+##########################################
+# OLD EKS CLUSTER
+---
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: common_environment_vars
+  annotations:
+    argocd.argoproj.io/hook: PreSync
+    argocd.argoproj.io/sync-wave: "-3"
+spec:
+  backendType: systemManager
+  data:
+  {{- range $varName, $relPath := .Values.parameters }}
+    - key: "{{ .Values.ssmPath }}{{ $relPath }}"
```

Comment:
For `.Values.ssmPath` here do you need to reference it via `$root`? I thought due to the context change in the loop/range using `$root.Values.ssmPath`. Same goes for line 37

Reply:
I'll check on that - it seemed to work on the reference test we did, but am double checking ...

Reply:
`.Values.ssmPath` is missing the leading `$`

```
$.Values.ssmPath
```

Reply:
fixed